test/Raven/Tests/ClientTest.php

```diff
@@ -664,20 +664,14 @@ public function testCaptureExceptionHandlesExcludeOption()
 
     public function testCaptureExceptionInvalidUTF8()
     {
-        if (version_compare(PHP_VERSION, '7.0', '>=')) {
-            if (ini_get('zend.assertions') != 1) {
-                $this->markTestSkipped('Production environment does not execute asserts');
-                return;
-            }
-        }
         $client = new Dummy_Raven_Client();
         try {
             invalid_encoding();
         } catch (Exception $ex) {
             $client->captureException($ex);
         }
         $events = $client->getSentEvents();
-        $this->assertEquals(1, count($events));
+        $this->assertCount(1, $events);
```

Comment:
There was no reason to do it. We have a lot of $this->assertEquals(1, count($events)); we should fix all of them in one step